actionpack/lib/abstract_controller/caching/fragments.rb

```diff
@@ -86,10 +84,8 @@ def read_fragment(key, options = nil)
         return unless cache_configured?
 
         key = fragment_cache_key(key)
-        instrument_fragment_cache :read_fragment, key do
```

Comment:
Sorry, "remove" was ambiguous wording, I should have said disable.

People should still be able to log fragment caching, we just don't want it on for new apps in Rails 5.1.

We need a config option on Action View, say, `enable_fragment_cache_logging`. It's important that we don't muck with the instrumentation, that is useful for benchmarks and data aggregation, but just turn off the logging within it.

If that flag is off, we set the cache store logger to nil to return early here https://github.com/rails/rails/blob/62514c5db76f77afc617cd02981a68abbcf58e87/activesupport/lib/active_support/cache.rb#L566

@dhh does that sound good to you?